web/crux/assets/install-script/install-k8s.sh.hbr

```diff
@@ -45,18 +45,13 @@ metadata:
   labels:
     rbac.dyrector.io/crane: "restricted-api-access"
 rules:
--
-  apiGroups:
-  - apps
-  - ""
-  - "monitoring.coreos.com"
+- apiGroups: [""]
```

Comment:
is this duplicated?

Reply:
Where is it duplicated? I could not find the exact same line anywhere else.
It refers the kubernetes-core API group.